snapcraft/commands/chassis.start

```diff
@@ -22,10 +22,12 @@ done
 export PATH="${OVN_RUNDIR}/bin/:${PATH}"
 
 # Prepare the arguments
+# By specifying "--db-*-create-insecure-remote=no" we prevent creation of
+# hardcoded bindings and we can use database to configure remotes later.
 OVN_ARGS="--db-nb-addr="${OVN_LOCAL_IP}" \
---db-nb-create-insecure-remote=yes \
```

Comment:
Not a problem of this patch, but why do we even list these arguments when starting the controller?

AFAICT none of these are required as the `ovn-controller` will get its information from the local Open vSwitch DB

Reply:
I think you are right. I'll take them out and test that everything works as expected.